x-pack/plugins/ml/public/maps/anomaly_layer_wizard.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { LAYER_WIZARD_CATEGORY } from '../../../maps/common';
+import type { LayerWizard } from '../../../maps/public';
+
+export const anomalyLayerWizard: Partial<LayerWizard> = {
+  categories: [LAYER_WIZARD_CATEGORY.SOLUTIONS],
+  description: i18n.translate('xpack.ml.maps.anomalyLayerDescription', {
+    defaultMessage: 'Create anomalies layers',
+  }),
+  disabledReason: i18n.translate('xpack.ml.maps.anomalyLayerUnavailableMessage', {
+    defaultMessage:
+      'Whatever reason the user cannot see ML card (likely because no enterprise license or no ML privileges)',
+  }),
+  icon: 'outlierDetectionJob',
+  getIsDisabled: () => {
+    // return false by default
```

Comment:
Consider having this check for enterprise license. For an example, see the track layer wizard.

https://github.com/elastic/kibana/blob/0675a6a48c94c030e93b007dbce55c0b532659ad/x-pack/plugins/maps/public/classes/sources/es_geo_line_source/layer_wizard.tsx#L25-L29

Reply:
This is checked in the plugin setup and passed through to the layer wizard https://github.com/elastic/kibana/pull/122862/files#diff-9658f57fcb39c79f75a675c515053ccb46c515a7e9f9eac7ecb200d21c3af09aR172

This default is just a placeholder.

Reply:
ah makes sense, thx! (I missed that)

fwiw: you could still register the card, but in "disabled" state. That way the card is still visible (but in disabled grayed-out state) . The advantage is that this feature would still be advertized for non-enterprise users.